src/clientsettingsdlg.cpp

```diff
@@ -396,6 +419,124 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient*         pNCliP,
 
     UpdateSoundCardFrame();
 
+    // Add help text to controls -----------------------------------------------
+    // Musician Profile
+    QString strFaderTag = "<b>" + tr ( "Musician Profile" ) + ":</b> " +
```

Comment:
I don't like the variable name, it appears to be c-n-p from elsewhere...

Reply:
I'd also prefer finer-grained What'sThis coverage - i.e. each field should have its own.  The second paragraph given here would be appropriate for the whole dialogbox, though, although the phrasing is a little confusing.

So, something like this might be better (following the style guide for the Accessibility labels vs the Whats This labels, if we have one).

- Dialog: The detail set here are shown in the server list when connecting to a server and in the channel label when you connect to a server.  Use them to provide some information about yourself.

- Alias: your name or the name you choose to perform under.

- Instrument: the instrument you play or the vocal part you perform.  Displayed as a picture in the client.

- Skill: your chosen skill rating.  Displayed as the background colour in the client.

- City: the city you're performing from.

- Country: where you're performing from.  Displayed as a flag in the client.